funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -562,7 +562,6 @@ def cli_run():
 
     try:
         global logger
-        # TODO Update logger to use the RotatingFileHandler in the funcx.utils.loggers.set_file_logger
```

Comment:
Do not remove this comment unless it has actually been implemented. Right now the options for RotatingFileHandler may not be passed all the way down, and once that is working we need those to be set here.

Reply:
Update this comment to say that the config options for the rotatingfilehandler need to be implemented and checked so that it is user configurable.